### ISSUE TITLE
[HOLD] [$1000] Web - Closing Connect bank account drawer doesn't show the right URL @aneequeahmad

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#10939]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open web app https://new.expensify.com/
2. Click on user icon on LHN to open setting.
3. Click on user's workspace.
4. Click on Connect bank account.
5. Click on close icon on top right.

## Expected Result:
Url in browser should be r/reportId

## Actual Result:
Url dispalyed as workspace/workspaceId or https://new.expensify.com/settings

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.99.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/189695554-19731cac-b32b-42d5-b57c-402180c6fcc7.mov

**Expensify/Expensify Issue URL:** 

**Issue reported by:** @aneequeahmad 

**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1661274757727099

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by abekkala** [comment]
I was able to reproduce

**Comment by kbecciv** [comment]
@mdneyazahmad Corrected reporter

**Comment by Gonals** [comment]
Setting as external!

**Comment by grgia** [comment]
Setting to external. Note similar issues linked above

**Comment by JmillsExpensify** [comment]
@Gonals Do I need to re-apply the `External` label here? I'm still not certain on the new process.

**Comment by Gonals** [comment]
Nope, I think "exported" is the way to go

**Comment by JmillsExpensify** [comment]
We still haven't received proposals, and in addition, Melvin hasn't updated the issue title with the amount. I'll do both now to try and get more visibility on this one.

**Comment by JmillsExpensify** [comment]
Still waiting for proposals.

**Comment by JmillsExpensify** [comment]
Going to bump this issue up to $1000 to try and get more eyes on this one.

**Comment by parasharrajat** [comment]
@JmillsExpensify There is no Engineer assigned to this issue. Secondly, we should hold this issue with others and add it to the sequence of the steps in #11111 's description.

**Comment by JmillsExpensify** [comment]
Yep. @Gonals Are you not on CME and that's why you unassigned yourself?

**Comment by JmillsExpensify** [comment]
>add it to the sequence of the steps in https://github.com/Expensify/App/issues/11111 's description.

Then @parasharrajat I'm not sure what you mean by this. The reproduction steps for this issue appear in 1111s description, and this issue is also linked. Let me know if I'm missing something. I'll go ahead and put a hold though in the meantime.

**Comment by parasharrajat** [comment]
If they are already added, then we are good. Thanks

**Comment by JmillsExpensify** [comment]
Ok cool, yes we're good! Will wait to see what Alberto says about removing himself while we're on Hold.

**Comment by Gonals** [comment]
Ah, I can take it, but I got assigned to this PR during triage, as a triage engineer, not as a CME, which is why I unassigned myself. The external label later should have assigned another engineer (or myself, if that's what got rolled), but I think this issue got there right when we were deprecating the `Exported` label, so it didn't get en Engineer with `External` and it seems it missed getting one with `Exported`.

TLDR: This does need an engineer and it can be me or we can try re-applying `External` and see what happens 🤷

**Comment by JmillsExpensify** [comment]
All good! I'll just re-apply the `External` label. We're still on hold in any case, so there aren't any immediate next steps right now.

**Comment by chiragsalian** [comment]
Issue is on HOLD so nothing actionable here atm. Demoting to weekly.

**Comment by JmillsExpensify** [comment]
Thanks, also still working on next steps in the tracking issue. We're still in a holding position until we sort that out.

**Comment by JmillsExpensify** [comment]
Still working out next steps in the tracking issue.

**Comment by JmillsExpensify** [comment]
Still on hold Melvin.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Still held as the linked issue is going through the design process.

**Comment by JmillsExpensify** [comment]
Still on hold per the linked issue.

**Comment by JmillsExpensify** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Still held.

**Comment by JmillsExpensify** [comment]
Still held on the linked issue as well as the navigation re-write.

**Comment by JmillsExpensify** [comment]
Still on hold for navigation. No ETA.

**Comment by JmillsExpensify** [comment]
Same same.

**Comment by JmillsExpensify** [comment]
Still on HOLD.

**Comment by JmillsExpensify** [comment]
Same as above.

**Comment by JmillsExpensify** [comment]
Alright, I'm circling back to this testing this issue. Here's my summary:

 - We've deprecated the close button, so that's not longer a relevant part of the testing steps
 - Further, when I go from the workspace editor, to bank account, and then back to workspace, the URLs are correct each time. I go from https://new.expensify.com/workspace/ and then to https://new.expensify.com/bank-account/ before going back to the right workspace URL. 

Accordingly, I think we're good to go here and we'll pay out @aneequeahmad $250 for reporting.

**Comment by JmillsExpensify** [comment]
Putting this back on hold purely for internal prioritization. Still working through the remaining issues.

**Comment by JmillsExpensify** [comment]
Same

**Comment by chiragsalian** [comment]
not overdue, on hold

**Comment by JmillsExpensify** [comment]
@aneequeahmad I think we should be good on this issue, though can you test everything up to step 5 in the reproduction steps and confirm that all the URLs/paths are correct?

**Comment by aneequeahmad** [comment]
@JmillsExpensify i think we are good to close this issue as this issue doesn’t exist anymore in production.

**Comment by JmillsExpensify** [comment]
Please share a video confirming in the future. I'll approve this first one. Offer sent for reporting this one.

**Comment by aneequeahmad** [comment]
@JmillsExpensify Sure, ill share the video in future. Accepted the offer that you sent

**Comment by JmillsExpensify** [comment]
This one is all paid out. Thanks!
